scripts/call_get_counter.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sncast call \
+    --contract-address 0x033c9817a4f4f7941451295fbcd5f34c589115af8716991f21a6afab5b3e3b57 \
+    --function "get_counter" \
```

Shellcheck (0.10.0)

[error] Couldn't parse this escaped char. Fix to allow more checks.

(SC1073)